examples/models/object_detection/faster_rcnn/train.py

```diff
@@ -63,7 +63,9 @@
 eval_ds = tfds.load("voc/2007", split="test", with_info=False)
 
 with strategy.scope():
-    model = keras_cv.models.FasterRCNN(classes=20, bounding_box_format="yxyx")
+    model = keras_cv.models.FasterRCNN(
+        classes=20, bounding_box_format="yxyx", include_rescaling=True
```

Comment:
I thought the training script uses `tf.keras.applications.resnet_50.preprocess_input`?

Reply:
Yes -- switched this to `False`